Fixes #830 - Update TopSky colours and settings

SamLefevre opened the pull request:
Fixes #830 

# Summary of changes

Updated TopSky colours to reflect closer to RW systems.

# Screenshots (if necessary)

## NODE

![image](https://github.com/user-attachments/assets/9e3c6150-0430-4156-bdbf-586f1f2706cf)

## NOVA

![image](https://github.com/user-attachments/assets/90ce32e6-d121-4f18-951e-bbe7b2239e2f)

## NERC

![image](https://github.com/user-attachments/assets/88c57563-49d1-4374-9b6b-6b806d7d78ed)

SamLefevre commented:
@luke11brown while here can we change TC non-concerned back to the old green? Also suggesting changing level to default main colour to indicate handoff, lost that functionality with CCDS-R
![image](https://github.com/user-attachments/assets/6fe9a4f9-ca2c-48cd-bf65-5e67dd824ee5)

luke11brown commented:
> @luke11brown while here can we change TC non-concerned back to the old green? Also suggesting changing level to default main colour to indicate handoff, lost that functionality with CCDS-R.

Sounds good

SamLefevre commented:
# Settings changes
## Colours
As above changed colours, also made CPDLC messages on lighter backgrounds darker blue for readability (like on iTEC).
![image](https://github.com/user-attachments/assets/681345f9-dabc-4442-85af-d1578dafc0d1)
## Assume Route draw
Disabled on NODE/NOVA.
## Alert List
Moved closer to UKCP ASR/MSL lists, was previously on top of the Sector Inbound List in the way.
![image](https://github.com/user-attachments/assets/50e64ab1-fa47-4831-9d36-48022fdad9fb)
## RDF
Disabled on NODE/NOVA.
## CARD
Disabled on NOVA.
## CPDLC and other windows with fixed-width fonts
To stop size of windows changing and overlapping symbols on CPDLC window, now uses Consolas instead of default EuroScope font.
### EuroScope custom font (DK-COOPANS, not fixed-width)
![image](https://github.com/user-attachments/assets/de624db7-58ce-45cc-8fc9-70c7d03c73c1)
### Consolas
![image](https://github.com/user-attachments/assets/f71dec21-7ae8-4279-b7f0-a75dde517ea5)
## CPDLC Window
Disabled on NODE/NOVA and moved closer to shortcut window on NERC/iTEC.
![image](https://github.com/user-attachments/assets/3fb07b1e-1ce7-4c39-8198-29f5e06a2d79)
